src/gameobjects/ball.js

```diff
@@ -33,32 +35,43 @@ export class Ball extends createjs.Bitmap {
       this.y = GAMEPLAY.HEIGHT - bounds.height / 2 - 1;
       this.velY *= -1;
 
-      if (!this.hitSound) {
-        // get sound instance
-        this.hitSound = createjs.Sound.play("bounce");
-        this.hitSound.volume = this.game.sfxVolume; // <-- apply current SFX volume;
-      } else {
-        this.hitSound.volume = this.game.sfxVolume;
-        this.hitSound.play();
-      }
+      this.playSound("bounceSound", "bounce");
     }
   }
 
-  changeColor(e) {
+  handleClick(e) {
     const _ball = e.target;
-    _ball.filters = [
+
+    _ball.changeColor(_ball);
+    _ball.playSound("clickSound", "click");
+    _ball.clicked = true;
+  }
+
+  playSound(soundName, file) {
+    if (!this[soundName]) {
+      // get sound instance
+      this[soundName] = createjs.Sound.play(file);
+      this[soundName].volume = this.game.sfxVolume; // <-- apply current SFX volume;
+    } else {
+      this[soundName].volume = this.game.sfxVolume;
+      this[soundName].play();
+    }
+  }
```

Comment:
thinking of making this a global function since I use the same logic in other files